### ISSUE TITLE
fastify@4.x deprecation notice (FSTDEP012) "Request#context will be removed in `fastify@5`

### ISSUE THREAD

**Issue by freakyfelt** [seriousme/fastify-openapi-glue#472]
I'm playing with fastify-openapi-glue and noticed a deprecation warning is emitted when accessing Request#context noting that it will be removed in fastify 5.x. Are there plans for addressing that and, if so, anything people can do to help?

```
(node:43644) [FSTDEP012] FastifyDeprecation: Request#context property access is deprecated. Please use "Request#routeConfig" or "Request#routeSchema" instead for accessing Route settings. The "Request#context" will be removed in `fastify@5`.
```

**Comment by freakyfelt** [comment]
Actually since this is only coming up when accessing the `route.context.config` it might just be updating the docs to use `route.routeConfig`

**Comment by seriousme** [comment]
Hi,

thanks for reporting this. I have updated the docs and the two tests that relied on context.schema.

Kind regards,
Hans

**Comment by seriousme** [comment]
FYI: Released as v4.1.4
